compiler/rustc_mir_transform/src/coverage/counters.rs

```diff
@@ -1,4 +1,7 @@
-use std::fmt::{self, Debug};
+use std::{
+    collections::BTreeMap,
+    fmt::{self, Debug},
+};
```

Comment:
Nitpick: I usually have a separate import for each module, so `collections` and `fmt` should be separate imports.